cronet/README.md

```diff
@@ -17,13 +16,26 @@ Some advantages of using Cronet with gRPC:
 * Robust to Android network connectivity changes
 * Support for [QUIC](https://www.chromium.org/quic)
 
-Cronet jars are available on Google's Maven repository. See the example app at
-https://github.com/GoogleChrome/cronet-sample/blob/master/README.md. To use
-Cronet with gRPC, you will need to copy the gRPC source files contained in this
-directory into your application's code, as we do not currently provide a
-`grpc-cronet` dependency.
+Since gRPC's 1.24 release, the `grpc-cronet` package provides access to the 
+`CronetChannelBuilder` class. Cronet jars are available on Google's Maven repository. 
+See the example app at https://github.com/GoogleChrome/cronet-sample/blob/master/README.md.
 
-To use Cronet, you must have the `ACCESS_NETWORK_STATE` permission set in
+## Example usage:
+
+In your app module's `build.gradle` file, include a dependency on both `grpc-cronet` and the 
+Google Play Services Client Library for Cronet
+
+```
+implementation 'io.grpc:grpc-cronet:1.24.0'
+implementation 'com.google.android.gms:play-services-cronet:16.0.0'
+```
+
+In cases where Cronet cannot be loaded from Google Play services, there is a less performant 
+implementation of Cronet's API that can be used. Depend on `org.chromium.net:cronet-fallback` 
+to use this fall-back implementation,  
```

Comment:
Period?

Reply:
It's a period. Changed. Thanks for catching it.